src/test_fixture/net.rs

```diff
@@ -1,4 +1,4 @@
-use crate::http::discovery::conf::Conf;
+use crate::net::discovery::conf::Conf;
```

Comment:
a small nit - we can re-export `Conf` from `discovery` mod so import does not have `conf` repeatedly

Reply:
done